### PR TITLE
for adaptive course, don't show lifes in popinend 

### PR DESCRIPTION
**Pull request by esa-coorp**
Before:
Capture d’écran 2019-09-02 à 09 54 55
after:
Capture d’écran 2019-09-02 à 09 55 32

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [x] Manual testing
- [x] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/feedback/index.js [hunk 1]
```diff
@@ -9,7 +12,7 @@ const Feedback = (props, context) => {
   const resource = media &&
     media.type && {
       ...media,
-      ...getOr({}, 'src.0', media)
+      ...pipe(get('src.0'), extend({description: mediaDescription}))(media)
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-components/src/molecule/feedback/index.js]: why do you need to change this logic whereas it's only to remove lives in popin-end header ?

File: packages/@coorpacademy-player-store/src/utils/state-extract.js [hunk 2]
```diff
@@ -237,16 +237,30 @@ export const getPrevStepContent = (state: State): Content | void => {
...
+  if (progression.state.nextContent.type === 'slide') {
+    return getChapterIdBySlide(state, progression.state.nextContent.ref);
+  } else {
+    if (!progression.state.slides || isEmpty(progression.state.slides)) {
+      return;
+    }
+    const lastSlideId = last(progression.state.slides);
+    return getChapterIdBySlide(state, lastSlideId);
+  }
+};
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-player-store/src/utils/state-extract.js]: same for this

Comment (godu) [reply packages/@coorpacademy-player-store/src/utils/state-extract.js]: => https://github.com/CoorpAcademy/components/pull/1617/files#diff-66e631ed454627bf02a2067a526bf492R803-R810
C'est dans le cas où on est sur un extraLife, le nextContent n'est plus la slide.

Comment (esa-coorp) [reply packages/@coorpacademy-player-store/src/utils/state-extract.js]: `popinEnd` uses the `getLives` function to determine whether to show/hide life

=> https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-player-web/src/map-state-to-props/popin-end.js#L79

=> https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-player-store/src/utils/state-extract.js#L460

File: packages/@coorpacademy-components/src/molecule/feedback/index.js (hunk 1 above)
Comment (esa-coorp) [reply packages/@coorpacademy-components/src/molecule/feedback/index.js]: for a popinEnd with pdf ressource, the description of the resource was not displayed

File: packages/@coorpacademy-player-store/src/utils/state-extract.js (hunk 2 above)
Comment (VincentCATILLON) [reply packages/@coorpacademy-player-store/src/utils/state-extract.js]: ok 👍

**Pull request merged**